<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab 2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelPicture">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectPicture">
        <property name="text">
         <string>Выбрать изображение</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="Histogram" name="histogramRed" native="true">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Histogram" name="histogramGreen" native="true">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Histogram" name="histogramBlue" native="true">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Histogram</class>
   <extends>QWidget</extends>
   <header>histogram.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
